Mathlib/Algebra/Order/GroupWithZero/Action/Synonym.lean

```diff
@@ -0,0 +1,70 @@
+/-
+Copyright (c) 2021 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Order.GroupWithZero.Synonym
+import Mathlib.Algebra.SMulWithZero
+import Mathlib.Tactic.Common
+
+/-!
+# Actions by and on order synonyms
+-/
+
+variable {G₀ M₀ : Type*}
+
+namespace OrderDual
+
+instance instSMulWithZero [Zero G₀] [Zero M₀] [SMulWithZero G₀ M₀] : SMulWithZero G₀ᵒᵈ M₀ :=
```

Comment:
Is the `SMul` instance defined in an earlier file? If so, can you mention where that instance is (and the lemmas around it) in the docstrings of the new files?

Reply:
`Algebra.Order.Group.Action.Synonym` is where it is defined